### PR TITLE
Added NLTK similarity

### PR DESCRIPTION
**Pull request by mekomlusa**
Rouge 1 results
Avg. P of 5 samples in rounge 1: 0.24352707749766575
Avg. R of 5 samples in rounge 1: 0.16800105165112883
Avg. F-1 of 5 samples in rounge 1: 0.19434407402918988
Rouge 2 results
Avg. P of 5 samples in rounge 2: 0.07655917666459214
Avg. R of 5 samples in rounge 2: 0.05580575909343033
Avg. F-1 of 5 samples in rounge 2: 0.06295466642369298

Incredibly slow though.

### CODE REVIEW

File: codes/extractive/netsum/netsum.py [hunk 1]
```diff
@@ -185,6 +196,48 @@ def data_prep(text_path, summary_path, news_words_path, is_lemmatize, file_info,
...
+        sentence (str): a string containing a collection of words.
+        
+    Return:
+        wordSense (list): a list of extracted sense (default to be the first one)
+    '''
+    wordSense = []
+    for w in re.findall(r'[a-zA-Z]+', sentence):
+        if w in STOP_WORDS or len(w) < WORD_MIN_LEN:
+            continue
+        senses = wn.synsets(w)
```
Comment (melonskin) [inline codes/extractive/netsum/netsum.py:209]: I tried to add pos='v' here and it kinds of hurt the performance. So we can do this for now.

File: codes/extractive/netsum/netsum.py [hunk 2]
```diff
@@ -185,6 +196,48 @@ def data_prep(text_path, summary_path, news_words_path, is_lemmatize, file_info,
...
+    Parameters:
+        wordSense1 (list): a list of extracted sense for the first sentence.
+        wordSense2 (list): a list of extracted sense for the second sentence.
+        
+    Return:
+        the similarity score (float).
+    '''
+    similarity = 0.0
+    total = 0.0
+    for sense1 in wordSense1:
```
Comment (melonskin) [inline codes/extractive/netsum/netsum.py:231]: check out the new version in lex rank, I use a dict to count senses to avoid duplicate work. Hope it helps.

**Review by melonskin — APPROVED** [review]

File: codes/extractive/netsum/netsum.py (hunk 1 above)
Comment (mekomlusa) [reply codes/extractive/netsum/netsum.py:209]: Great.

File: codes/extractive/netsum/netsum.py (hunk 2 above)
Comment (mekomlusa) [reply codes/extractive/netsum/netsum.py:231]: Will do in the next PR, thanks!

**Pull request merged**